src/include/storage/storage_defs.h

```diff
@@ -55,7 +67,8 @@ struct alignas(common::Constants::BLOCK_SIZE) RawBlock {
   /**
    * Contents of the raw block.
    */
-  byte content_[common::Constants::BLOCK_SIZE - 2 * sizeof(uint32_t) - sizeof(BlockAccessController)];
+  byte content_[common::Constants::BLOCK_SIZE - sizeof(uintptr_t) - sizeof(layout_version_t) - sizeof(uint32_t) -
```

Comment:
I think this under-calculates the header by 16 bits? Since you redefined layout version to be 16, but didn't explicitly add the padding here?

Reply:
You're right, we still want the old calculation so we can use the extra 16 bits for flags and such.